access.lisp

```diff
@@ -320,7 +321,10 @@
    if lax? we will ignore packages to find the slot we will always return a
    slot-name from the specified package if it exists, otherwise we return the
    slot-name we found if its in a different package"
-  (unless (and o (typep o '(or standard-object condition)))
+  ;; The accessing of structure objects slots is undefined behaviour by
+  ;; the language specification, but most Lisp implementations define
+  ;; slot-value on structure-objects in a meaningful and expected way.
```

Comment:
Maybe list the implementations here? The simple test of `(progn (defstruct foo (bar 42)) (slot-value (make-foo) 'bar))` returns `42` on current SBCL, CCL, ECL, ABCL, CLISP, Clasp, LW, ACL.